.gitignore

```diff
@@ -13,9 +13,6 @@ hack/tools/bin
 # E2E test templates
 test/e2e/data/infrastructure-docker/**/cluster-template*.yaml
 
-# E2e test extension deployment
-test/e2e/data/test-extension/deployment.yaml
```

Comment:
This file wasn't used since we were installing the test extension via clusterctl